sdk/keyvault/azure-security-keyvault-keys/pom.xml

```diff
@@ -46,12 +46,6 @@
       <version>1.1.0</version> <!-- {x-version-update;com.azure:azure-core-http-netty;dependency} -->
     </dependency>
 
-    <dependency>
-      <groupId>commons-codec</groupId>
-      <artifactId>commons-codec</artifactId>
-      <version>1.13</version> <!-- {x-version-update;commons-codec:commons-codec;external_dependency} -->
```

Comment:
should we remove this library from the external_dependencies.txt too?

Reply:
need to check if any other SDK still has this dependency.
Best to clean that up in a follow up PR.